instrumentation/opentelemetry-instrumentation-aiohttp-server/src/opentelemetry/instrumentation/aiohttp_server/__init__.py

```diff
@@ -238,14 +238,22 @@ def _parse_active_request_count_attrs(req_attrs):
 
 
 def get_default_span_name(request: web.Request) -> str:
-    """Default implementation for get_default_span_details
+    """Returns the span name.
     Args:
         request: the request object itself.
     Returns:
-        The span name.
+        The canonical name of a resource if possible or just request path.
```

Comment:
Nit: Needs an update


```suggestion
        The span name as "{method} {canonical_name}" of a resource if possible or just "{method}".
```